<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WatchList</class>
 <widget class="QDialog" name="WatchList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>415</width>
    <height>232</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Watch List</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>401</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QPushButton" name="okButton">
      <property name="text">
       <string>Add/Modify</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="delButton">
      <property name="text">
       <string>Delete</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="exportButton">
      <property name="text">
       <string>Export Now</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="closeButton">
      <property name="text">
       <string>Close</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>405</width>
     <height>183</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Watch List Number #</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Title</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Results Type</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Average results</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>From X:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>From Y:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>From Step</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLineEdit" name="watchIndexLine"/>
      </item>
      <item>
       <widget class="QLineEdit" name="titleLine"/>
      </item>
      <item>
       <widget class="QComboBox" name="watchCombo">
        <item>
         <property name="text">
          <string>Nodal Solution - X displacement</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Nodal Solution - Y displacement</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Nodal Solution - Excess pore pressure</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Element Solution - Effective Stress - Sxx</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Element Solution - Effective Stress - Syy</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Element Solution - Effective Stress - Sxy</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Element Pore Pressure </string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Element Solution - Total Stress - Sxx</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Element Solution - Total Stress - Syy</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Element Solution - Total Stress - Sxy</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="averageCheck">
        <property name="text">
         <string>Yes</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLineEdit" name="x0Line"/>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>To</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="x1Line"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLineEdit" name="y0Line"/>
        </item>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>To</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="y1Line"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLineEdit" name="beginStepLine"/>
        </item>
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>To</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="endStepLine"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
